grammar.js

```diff
@@ -775,17 +775,19 @@ module.exports = grammar({
       '}'
     ),
 
-    escape_sequence: $ => token(seq(
+    escape_sequence: $ => prec(1, token(seq(
```

Comment:
Is this `prec(1)` needed here? I would think that it would actually work without the precedence, because in order to resolve ambiguities at the token level, Tree-sitter [applies the longest-match rule](http://tree-sitter.github.io/tree-sitter/creating-parsers#conflicting-tokens).

Reply:
If I remove `prec(1)` then tests are failing.

Reply:
Oh yeah, I guess `prec(1, token(x))` behaves the same as `token(prec(1, x))` - the precedence is applied at the token level.